packages/cli/src/util/client.ts

```diff
@@ -0,0 +1,46 @@
+import { FetchLike, MedplumClient } from '@medplum/core';
+import { FileSystemStorage } from '../storage';
+
+export interface MedplumClientCommandOptions {
```

Comment:
Is this interface necessary, or could we just use `MedplumClientOptions`?

Reply:
This interface is not necessary. Another idea was to inherir from `MedplumClientOptions`, to include `command` specific global options such as `clientId` and `clientSecret`. `clientId` and `clientSecret` could also be added to `MedplumClientOptions` to be used more broadly by the SDK.